webhook_to_fedora_messaging_messages/forgejo/v1.py

```diff
@@ -0,0 +1,78 @@
+# SPDX-FileCopyrightText: 2024 Contributors to the Fedora Project
+#
+# SPDX-License-Identifier: LGPL-3.0-or-later
+
+import typing
+import uuid
+
+from ..base import Webhook2FedMsgBase
+from .utils import summarize_repository_event
+
+
+class ForgejoMessageV1(Webhook2FedMsgBase):
+    @property
+    def app_name(self):
+        return "Forgejo"
+
+    @property
+    def target_id(self):
+        return self.body["repository"]["id"]
+
+    @property
+    def signature(self):
+        return hashlib.sha1(str(self.body).encode(encoding="utf-8")).hexdigest()
+
+    @property
+    def delivery(self):
+        return self.body["headers"]["x-forgejo-delivery"]
+
+    @property
+    def signature_sha256(self):
+        return hashlib.sha256(str(self.body).encode(encoding="utf-8")).hexdigest()
+
+    @property
+    def event_name(self):
+        return self.body["headers"].get("x-forgejo-event")
```

Comment:
The json schema guarantees that this key exists in the `headers` dict, you can drop the `.get()`